src/Symfony/Component/HttpFoundation/Session/Flash/FlashBag.php

```diff
@@ -165,16 +163,4 @@ public function clear()
     {
         return $this->all();
     }
-
-    /**
-     * Returns an iterator for flashes.
-     *
-     * @deprecated Will be removed in 3.0.
-     *
-     * @return \ArrayIterator An \ArrayIterator instance
-     */
-    public function getIterator()
-    {
```

Comment:
@FlorianLB you'll need to rebase this PR. Deprecation warnings were already merged in and it conflicts with yours.